srcpkgs/emptty/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'emptty'
+pkgname=emptty
+version=0.3.0
+revision=1
+build_style=gnu-makefile
+make_build_target=build
+make_install_target="install-all install-runit install-config"
+makedepends="pam-devel go gcc libX11-devel"
+depends="pam libX11"
+short_desc="Dead simple Display Manager running in CLI as TTY login"
+maintainer="xXR01I1Xx <xxr01i1xx@tuta.io>"
+license="MIT"
+homepage="https://github.com/tvrzna/emptty"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=fa8a6336e150f0f74813ccf71817228deed9e06f1c8362e41634def959ff6c2b
+nopie=yes
+
+post_install() {
+	vlicense LICENSE
```

Comment:
You will need to manually copy over the man page, pam config, program config and runit service here. The makefile is poorly written and I wouldn't advise using it. Target dependencies are not defined, so `install-manual` will fail without manually running the `build` target, which you **don't** want to do here.

`xbps-src` provides the `vinstall` function to copy the pam config, program config and runit script. You can use the `vman` function to copy over the manual page. After you install the runit script, you will also need to create a `supervise` link that Void packages expect:
```
ln -s /run/runit/supervise.emptty ${DESTDIR}/etc/sv/emptty/supervise
```